src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  constructor() {
```

Comment:
[FUNCTION NAMING] - The constructor of `Herbivore` should accept a `name` parameter to be passed to the `super` constructor. Without this, all `Herbivore` instances would be created without a name, which violates the task requirement that all animals must have a `name` property.